pykafka/simpleconsumer.py

```diff
@@ -816,7 +816,7 @@ def enqueue_messages(self, messages):
         :type messages: Iterable of :class:`pykafka.common.Message`
         """
         for message in messages:
-            if message.offset < self.last_offset_consumed:
+            if message.offset <= self.last_offset_consumed:
```

Comment:
(..... or even stricter: `message.offset` ought to always equal `self.next_offset` I suppose)